### PR TITLE
Add response property added, removed and changed checkers

### PR DESCRIPTION
**Pull request by blva**

### CODE REVIEW

**Review by ciprian-tibulca — APPROVED** [review]

File: checker/check-response-pattern-added-or-changed.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+					CheckModifiedPropertiesDiff(
+						mediaTypeDiff.SchemaDiff,
+						func(propertyPath string, propertyName string, propertyDiff *diff.SchemaDiff, parent *diff.SchemaDiff) {
+							patternDiff := propertyDiff.PatternDiff
+							if patternDiff == nil {
+								return
+							}
+
+							id := ResponsePropertyPatternChangedId
+							text := fmt.Sprintf(config.i18n(id), ColorizedValue(propertyFullName(propertyPath, propertyName)), ColorizedValue(patternDiff.To), ColorizedValue(patternDiff.From), ColorizedValue(responseStatus))
```
Comment (reuvenharrison) [inline checker/check-response-pattern-added-or-changed.go]: 'from' and 'to' are reversed

File: checker/localizations_src/en/messages.yaml [hunk 2]
```diff
@@ -161,5 +161,8 @@ response-optional-property-became-write-only: the response optional property %s
 response-optional-property-became-read-only: the response optional property %s became read-only for the status %s
 response-optional-property-became-not-write-only: the response optional property %s became not write-only for the status %s
 response-optional-property-became-not-read-only: the response optional property %s became not read-only for the status %s
+response-property-pattern-changed: the response property %s pattern changed from %s to %s for the status %s
+response-property-pattern-added: the response property %s pattern %s was added for the status %s
+response-property-pattern-removed: the response property %s pattern %s was removed for the status %s
```
Comment (reuvenharrison) [inline checker/localizations_src/en/messages.yaml]: I suggest this template instead:
the %s response property's pattern was changed from %s to %s for the response status %s

File: checker/check-response-pattern-added-or-changed.go (hunk 1 above)
Comment (blva) [reply checker/check-response-pattern-added-or-changed.go]: thanks, updated!

File: checker/localizations_src/en/messages.yaml (hunk 2 above)
Comment (blva) [reply checker/localizations_src/en/messages.yaml]: ack, will update the one from the previous PR as well

File: checker/check-response-pattern-added-or-changed_test.go [hunk 3]
```diff
@@ -24,7 +24,7 @@ func TestResponsePropertyPatternChanged(t *testing.T) {
 	require.Equal(t, checker.ApiChange{
 
 		Id:          "response-property-pattern-changed",
-		Text:        "the response property 'data/created' pattern changed from '^[a-z]+$' to '^(?:([a-z]+-)*([a-z]+)?)$' for the status '200'",
+		Text:        "the 'data/created' response property pattern was changed from '^[a-z]+$' to '^(?:([a-z]+-)*([a-z]+)?)$' for the status '200'",
```
Comment (reuvenharrison) [inline checker/check-response-pattern-added-or-changed_test.go]: Please change:
the %s response property
To:
the %s response property's


Like the other messages, such as:
the %s response property's maxLength was unset from %s for the response status %s

Comment (blva) [reply checker/check-response-pattern-added-or-changed_test.go]: thanks, done!

**Pull request merged**